hyprland-share-picker/main.cpp

```diff
@@ -85,7 +85,7 @@ int main(int argc, char* argv[]) {
     MainPicker w;
     mainPickerPtr = &w;
 
-    QSettings* settings = new QSettings("hypr", "hyprland-share-picker");
+    QSettings* settings = new QSettings("/tmp/hypr/hyprland-share-picker.conf", QSettings::IniFormat);
```

Comment:
`/tmp` will be purged after a reboot.

Why are you adamant on saving the size? Can't the user set a windowrule?

Reply:
Yes they can, but that doesn't mean the window title is gonna fit
You either dont use the picker enough to care or you use the picker enough to really really care

Sometimes the windows you want to capture have titles like
"Perfect World - Rainbow 1"
"Perfect World - Rainbow 2"
"Perfect World - Rainbow 3"

But sometimes
"Your Worst Nightmare - yourprojectname - Asset Editor ~/user/projects/yourprojectname/Assets/Models/Enviroment/LevelX/somelongassetname1.glb*"
"Your Worst Nightmare - yourprojectname - Asset Editor ~/user/projects/yourprojectname/Assets/Materials/Enviroment/LevelX/somelongassetname2.glb*"
"Your Worst Nightmare - yourprojectname -Asset Editor ~/user/projects/yourprojectname/Assets/Models/Enviroment/LevelX/somelongassetname3.glb*"

I used the /tmp because you only care about remembering when you are using it constantly and just setting a bigger fixed size in a windowrule doesn't all your necessities unless there is a rule to remember a window size

You dont really care if you use that picker 3 times in a day

This is the reason i notice https://github.com/hyprwm/xdg-desktop-portal-hyprland/issues/169 and thats why i constatly crash xdg-desktop-portal-hyprland, because i use a lot that dialog

If your really dont want that feature i'm ok removing it from the PR as i told you before i'm just gonna maintain the feature in my fork for personal use (no package publishing)

Reply:
fair enough. Let's keep it in /tmp then.